samples/drivers/espi/prj.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_ESPI=y
 CONFIG_STDOUT_CONSOLE=y
 CONFIG_PRINTK=y
+CONFIG_GPIO=y
```

Comment:
Looks like this sample also optionally uses GPIO

Reply:
addressed